src/run.ts

```diff
@@ -55,6 +55,23 @@ export function run(
       showResults(results)
     })
 
+  program
+    .command('down-to')
+    .argument('<migration-name>')
+    .description('Migrates down to the specified migration name. Specify "NO_MIGRATIONS" to migrate all the way down.')
+    .action(async (name) => {
+      let results
```

Comment:
Please type the variable

```suggestion
      let results: MigrationResultSet
```